doc/manuals/admin/database_model.md

```diff
@@ -178,40 +178,40 @@ Fields:
 Example document:
 
 ```
- {
```

Comment:
JSON hasn't changed actually, but for some reason github shows this as change...

NTC